sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -140,6 +141,8 @@ def tune(
         parameters: Dict[str, Any],
         base_image: str = constants.BASE_IMAGE_TENSORFLOW,
         namespace: Optional[str] = None,
+        env: Union[Dict[str, str], List[V1EnvVar], None] = None,
+        env_from: Union[V1EnvFromSource, List[V1EnvFromSource], None] = None,
```

Comment:
Should we keep the name associated with Trials ? E.g. user could mislead this as environment for Algorithm Service.
E.g. name it as `trials_env`.
Also, I think we can allow to set `Dict[str, str]`,  `List[V1EnvVar]`, or `List[V1EnvFromSource]` in this parameter.

WDYT @shipengcheng1230 @tenzen-y @johnugeorge ?

Reply:
I like `env_per_trial` which is more descriptive and consistent with `resources_per_trial`!

Reply:
Yeah, I think so @shipengcheng1230, we can accept multiple types for `env_per_trial` parameter.
What do you think about this name @tenzen-y @johnugeorge ?

Reply:
Here is the tentative implementation 12eea643d8ba49fd28e9be354941f80687b842e1

Reply:
`env_per_trial` looks great!